node/src/reactor/joiner.rs

```diff
@@ -503,11 +505,8 @@ impl reactor::Reactor for Reactor {
         let block_header_by_hash_fetcher: Fetcher<BlockHeader> =
             Fetcher::new("block_header_by_hash", config.fetcher, registry)?;
 
-        let deploy_acceptor = DeployAcceptor::new(
-            config.deploy_acceptor,
-            &*chainspec_loader.chainspec(),
-            registry,
-        )?;
+        let deploy_acceptor =
+            DeployAcceptor::new(VERIFY_ACCOUNTS, &*chainspec_loader.chainspec(), registry)?;
```

Comment:
If it's not supposed to be changed, why even bother maintaining the option to do so via constructor? It adds confusion b/c tests should still be written for both cases (`true` and `false`).

Reply:
I've done it as the fetcher test uses a macro to instantiate its DA component, and it always goes via the constructor, and introducing the account verification for the gossiper/fetcher tests will cause them to fail, or require a rework of the test suites for those two components

Reply:
I wanted to remove it from the constructor as well, its just a constraint from a macro usage in the tests